src/ioctls/vcpu.rs

```diff
@@ -2644,6 +2644,22 @@ mod tests {
         assert!(faulty_vcpu_fd.translate_gva(u64::MAX).is_err());
     }
 
+    #[test]
+    #[cfg(target_arch = "aarch64")]
+    fn test_faulty_vcpu_target_aarch64() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        let vcpu = vm.create_vcpu(0).unwrap();
+
+        let mut kvi: kvm_bindings::kvm_vcpu_init = kvm_bindings::kvm_vcpu_init {
+            target: KVM_ARM_NUM_TARGETS,
+            ..Default::default()
+        };
+        kvi.target = KVM_ARM_NUM_TARGETS;
```

Comment:
I think `kvi.target = KVM_ARM_NUM_TARGETS;` is not needed here (also `mut` is not needed).
Additionally can you add a comment about `KVM_ARM_NUM_TARGETS` representing an incorrect target? Smth like: `"KVM defines valid targets from 0 to 5 and KVM_ARM_NUM_TARGETS has a value 6"`
Or maybe just use some big raw value here instead of a defined constant.